integration_tests/test_suites/k8s-test-suite/tests/test_integration.py

```diff
@@ -24,6 +25,35 @@
     get_test_project_environments_path,
 )
 
+from .utils import _wait_k8s_job_to_delete
+
+
+def _wait_k8s_job_to_start(
+    api_client: DagsterKubernetesClient, job_name: str, namespace: str
+) -> bool:
+    """Wait until Kubernetes job exists."""
+    for _ in range(5):
+        try:
+            api_client.batch_api.read_namespaced_job_status(job_name, namespace)
+            return True
+        except kubernetes.client.rest.ApiException:
+            time.sleep(5)
+
+    return False
+
+
+def _wait_job_termination_availability(
```

Comment:
nit: consider: _wait_until_job_can_be_terminated for readability

Reply:
Changed.